frontstage_api/controllers/case_controller.py

```diff
@@ -74,3 +91,57 @@ def post_case_event(case_id, party_id, category, description):
         logger.error('Failed to post to case service', case_id=case_id)
         raise ApiError(url, response.status_code)
     logger.debug('Successfully posted case event', case_id=case_id)
+
+
+def check_case_permissions(party_id, case_party_id, case_id=None):
+    logger.debug('Party requesting access to case', party_id=party_id, case_id=case_id, case_party_id=case_party_id)
+    if party_id != case_party_id:
+        logger.warning('Party does not have permission to access case', party_id=party_id, case_id=case_id, case_party_id=case_party_id)
```

Comment:
Do we need the repeat logging here and in the error handler?

Reply:
Nope, I'll remove